Dockerfile

```diff
@@ -48,6 +50,7 @@ RUN echo 'deb http://deb.debian.org/debian bullseye-backports main' \
       ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb && \
     apt-get update && \
     apt-get -y --no-install-recommends install libarrow-dev=9.0.0-1 libprotobuf-dev libparquet-dev=9.0.0-1 && \
+    python3 -m pip install --upgrade poetry && \
```

Comment:
we should validate that there is no conflict with the Poetry installed in https://github.com/tenzir/vast/pull/2625

Reply:
I verified this locally.